library_generation/model/config_change.py

```diff
@@ -156,3 +170,30 @@ def __create_qualified_commit(
         if len(libraries) == 0:
             return None
         return QualifiedCommit(commit=commit, libraries=libraries)
+
+    @staticmethod
+    def __qualified_build_change(commit: Commit, build_file_path: str) -> bool:
+        """
+        Checks if the given commit containing a BUILD.bazel change is a
+        qualified commit.
```

Comment:
Can you add another sentence explaining that the function parses the `gapic_inputs` from both versions to tell if there is a relevant change?

Reply:
Done.